modules/solid_mechanics/src/materials/SymmIsotropicElasticityTensor.C

```diff
@@ -68,6 +68,15 @@ SymmIsotropicElasticityTensor::shearModulus() const
 }
 
 Real
+SymmIsotropicElasticityTensor::youngsModulus() const
+{
+  if (!_E_set)
+    mooseError("Youngs modulus not set");
+
+  return _E;
+}
```

Comment:
We can make a material in the tensor mechanics model that tests it. It doesn't even have to be physical, just an object that exercises the interface. If that's a ton of new code to test or it creates a worthless object than I won't force it but I just want to always create tests when there's an opportunity to keep these interfaces from breaking in the future.